Src/CSharpier/CommandLineOptions.cs

```diff
@@ -23,6 +24,10 @@ public static RootCommand Create()
                     Arity = ArgumentArity.ZeroOrOne,
                     Description = "A path to a directory containing files to format or a file to format. If a path is not specified the current directory is used"
                 }.LegalFilePathsOnly(),
+                new Option(
```

Comment:
Have you considered using [System.CommandLine](https://github.com/dotnet/command-line-api/blob/main/docs/Your-first-app-with-System-CommandLine-DragonFruit.md)? It does some real _magic_!